.circleci/config.yml

```diff
@@ -32,11 +36,12 @@ jobs:
               else
```

Comment:
The line above needs to be removed and replaced with a call to the new push script.